README.md

```diff
@@ -8,15 +8,15 @@ For questions and discussion join the [Keras Slack](https://keras-slack-autojoin
 ## Setup
 
 1. Clone this repository
-1. `pip install .`
-1. `python setup.py build_ext --inplace`
+2. `pip install .`
+3. `python setup.py build_ext --inplace`
 
 ## Usage
 
 Basic usage:
 
 1. Define your own dataset in a csv format, for more information follow the guide in [keras-retinanet](https://github.com/fizyr/keras-retinanet#csv-datasets).
-1. Run `python optimize_anchors.py PATH_TO_CSV` 
+2. Run `anchor-optim PATH_TO_CSV` 
```

Comment:
Could we use a `anchor-optimization` instead, its just saving a few characters.

Reply:
Yes, I'll modify the command from 'anchor-optim' to 'anchor-optimization'